src/main/java/com/thealgorithms/maths/DudeneyNumber.java

```diff
@@ -32,11 +32,6 @@ public static boolean isDudeney(int n) {
             temp /= 10;
         }
 
-        //If the cube root of the number is not equal to the sum of its digits we return false.
```

Comment:
this comment line should be there. other than that everything looks good.

Reply:
Added back in even though it directly describes exactly what the code does. Is it not irrelevant anymore?

Reply:
yup you are right.